modules/apps/site/site-insurance-site-initializer/src/main/zippableResources/fragments/raylife/fragments/selected-quote/index.css

```
@@ -0,0 +1,18 @@
+.quote-comparison H1 {
```

Recomendo mudar o nome desta classe, por ser o mesmo nome usado no ID da tela anterior a essa. Podendo causar um problema de entendimento em algum dev que no futuro venha a fazer manutenção nas duas telas.

Poderia ser algo do tipo:
```suggestion
.quote-comparison-selected H1 {
```

Após a mudança, favor modificar em todos lugares que utilizarem este mesmo nome dentro do contexto da página que estiver trabalhando.